README.md

```diff
@@ -592,8 +595,119 @@ public Object findById(
 }
 ```
 
+Conversions
+-------------------
+
+Specification argument resolvers uses conversion mechanism to convert request string params to types of fields for which specifications have been defined.
+
+Let's consider the following code:
+  ```java
+	@Controller
+	public class SampleController {
+		
+		@Autowired
+		CustomerRepository customerRepository;
+		
+		@RequestMapping("/find")
+		public List<Customer> findByRegistrationDate(
+				@And({
+					@Spec(path = "name", params = "name", spec = Equal.class),
+					@Spec(path = "registrationDate", params = "registrationDate", spec = GreaterThanOrEqual.class)
+				}) Specification<Customer> spec) {
+			return customerRepository.findAll(spec);
+		}
+	}
+	
+	@Entity
+	public class Customer {
+		String name;
+		Date registrationDate;
+	}
+  ``` 
+When the following request will be sent to the endpoint presented above
+  ```
+  GET /find?name=John&registrationDate=2020-06-19
+  ```
+
+a `Specification<Customer>` based on fields `name` and `registrationDate` will be built.
+
+  * The type of the `name` field is a `String` type, received parameter value is always a `String` type so there is no need of conversion.  
+  * The type of the `registartionDate` field is `java.util.Date` type,  the parameter is a `String` type so `String` type will be converted into `Date` using one of available converter.
```

Comment:
The type of the `registartionDate` field is `java.util.Date`, so the (`String`) will be converted into `Date` using one of the available converters.